src/Symfony/Component/Console/Style/SymfonyStyle.php

```diff
@@ -163,6 +167,24 @@ public function text($message)
     /**
      * {@inheritdoc}
      */
+    public function comment($message)
+    {
+        $this->autoPrependText();
+
+        if (!is_array($message)) {
```

Comment:
Oh ! Actually I didn't see it was the same implementation as `text`... It would be a minor BC break again if someone rely on this strange behavior.
Would `string[]|string` be more accurate in the docblock ?